### ISSUE TITLE
Latest version 1.7.0 for Windows flagged with trojan

### ISSUE THREAD

**Issue by DeeSass** [trembon/switch-library-manager#51]
switch-library-manager.windows-amd64.1.7.0.zip

Windows Defender won't allow download, keeps detecting trojan

Trojan:Script/Wacatac.H!ml
Trojan:Script/Sabsik.FL.A!ml

I know false positives can happen but the previous version does not have this issue...

**Comment by trembon** [comment]
I have has this problem with every release, even before my fork.
When i tried to read up on it, it seems that parts of Go is flagged by MS and its not anything i can do about it.

Will see if i can find that source again

**Comment by harvestry-of-ghosts** [comment]
It's really weird for me. Sometimes MS Defender detects it and sometimes it doesn't. It happens across forks and across versions. Haven't been able to figure out why sometimes I can even do a manual scan on the exact same items and it says there's a trojan, but I can scan them again and there's nothing found even though I never added anything to exceptions. I have not yet had it tell me there's a trojan in 1.7.0. (1.6.0 did) .Not by downloading it, extracting the archive, running the app, or doing a manual scan. But from my experience it could just trigger MS Defender at any random time. False positive and a weird inconsistent issue with MS Defender.

**Comment by DeeSass** [comment]
Yes, I tried downloading it again and Defender did not trigger this time. I tried manually scanning and could not get it to trigger. Before I posted this Issue it triggered for me on 3 consecutive download attempts and a manual scan as well. Also noticed it identifies different scripts each time. Very strange! At this point I trust it’s a bug and a no longer concerned. Love this application by the way, thanks so much trembon for keeping it alive and carrying it forward.

**Comment by NikoMuffin** [comment]
I just wanted to post here telling @trembon how utterly grateful I am for him to continue this project. It's the only one that actually works flawlessly.

**Comment by trembon** [comment]
seems like this is mentioned on the official site here: https://go.dev/doc/faq#virus

but seems like it can be solved by signing the .exe file, so it can maybe be worth a try
https://www.reddit.com/r/golang/comments/s1bh01/goexecutables_and_windows_defender/
